tests/console/hostname.pm

```diff
@@ -28,6 +28,17 @@ sub run {
     script_run "systemctl --no-pager status network.service";
     save_screenshot;
     assert_script_run "if systemctl -q is-active network.service; then systemctl reload-or-restart network.service; fi";
+    if (script_run("ip addr show br0 | grep DOWN") == 0) {
+        record_soft_failure('bsc#1061051');
+        assert_script_run "systemctl reload network";
+        script_run "systemctl --no-pager status network.service";
```

Comment:
We have a `systemctl` wrapper command in `lib/utils`. Can you use that one please?